pkgs/development/tools/xcbuild/specs/Unifdef.xcspec

```diff
@@ -0,0 +1,32 @@
+/*
+    Unifdef.xcspec
+
+    Copyright (c) 2014 Apple, Inc.
+    All rights reserved.
```

Comment:
Not sure people will be very happy with an "All rights reserved" Apple file living in this repo 😦 

I'm guessing it doesn't exist anywhere else public? If so, I think our best bet is to shove all these files into a git repo somewhere that we fetch from the xcbuild expression.